azure/packages/test/end-to-end-tests/src/test/AzureClientFactory.ts

```diff
@@ -33,7 +33,7 @@ export function createAzureClient(
 		name: userName ?? uuid(),
 	};
 	const endPoint = process.env.azure__fluid__relay__service__endpoint as string;
-	if (endPoint === undefined) {
+	if (useAzure && endPoint === undefined) {
```

Comment:
Maybe just curiosity, but what does it mean for useAzure to be false here? Which service is used in that case?